docs/docs-content/release-notes/annoucements.md

```diff
@@ -36,6 +36,11 @@ below and take necessary actions to avoid any disruptions in your environment.
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------- |
 | The Terraform resource, `spectrocloud_cluster_import` is deprecated. To import a cluster deployed outside of the context of Palette, refer to the [Import a Cluster](../clusters/imported-clusters/cluster-import.md) guide. | January, 2025 | November 15, 2024 |
 
+| The Earthly build argument `PROXY_CERT_PATH` is deprecated. To provide client certificates needed for proxy servers,
```

Comment:
@lennessyy we have this in the release notes PR. I'm happy to change , but I think being more specific and mentioning EdgeForge and CanvOS helps the user a bit more 😅 

```
| The EdgeForge build process utility, CanvOS has an argument variable named `PROXY_CERT_PATH`. This variable is deprecated and no longer the recommended way to pass proxy certificates to the CanvOS build process. Use the **certs** folder in the root of the project directory to store proxy certificates. The **certs** folder is automatically included in the CanvOS build process. Refer to the [Build Provider Images](../clusters/edge/edgeforge-workflow/palette-canvos/build-provider-images.md) for guidance on using the **certs** folder to pass proxy certificates to the CanvOS build process. | March 15, 2025 | December 7, 2024  |

```